project_task_pull_request/models/project_task.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2017 Specialty Medical Drugstore
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, exceptions, fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+
+    pr_uri = fields.Char(string="PR URI", track_visibility="onchange")
+
+    pr_required_states = fields.Many2many(
+        related="project_id.pr_required_states", readonly=True
```

Comment:
Since v12 `readonly=True` is default, maybe you should remove it

Reply:
Well, it's not problem to explicitly put, but indeed can be removed